cscs-checks/microbenchmarks/gpu/gpu_burn/gpu_burn_test.py

```diff
@@ -24,30 +24,29 @@ class gpu_burn_check(GpuBurn):
     num_tasks = 0
     reference = {
         'dom:gpu': {
-            'perf': (4115, -0.10, None, 'Gflop/s'),
+            'min_perf': (4115, -0.10, None),
```

Comment:
```suggestion
            my_limits = "-0.10, None, 'Gflop/s'"
            ...
            'min_perf': (4115, my_limits),
```
Feel free to ignore

Reply:
nevermind

Reply:
I see what you mean there. I guess that one can still have the units as a comment in there instead of being part of the reference tuple. At the moment setting the reference tuple the "old way" when using this new syntax would throw an error. I think it's a good thing to throw an error, because otherwise people would use a bit of a hybrid syntax, which might lead to errors if at some point we drop the current syntax